install

```diff
@@ -28,7 +28,12 @@ build_clients=false
 build_release=true
 run_tests=false
 build_hip_clang=false
-rocm_path=/opt/rocm/bin
+
+rocm_path=/opt/rocm
+if ! [ -z ${ROCM_INSTALL_PATH+x} ]; then
+    rocm_path=${ROCM_INSTALL_PATH}
```

Comment:
How about adding a command-line option, say, --prefix, to specify where to install instead of relying on an undocumented environment variable?